pyquil/api/_compiler.py

```diff
@@ -354,8 +355,10 @@ def native_quil_to_executable(
 
         arithmetic_response = rewrite_arithmetic(nq_program)
 
+        nq_program_calibrated = self.expand_calibrations(nq_program)
```

Comment:
Alex had requested that we log the program text at debug level this point, so that a user of a higher-level program (of which pyquil is only a dependency) can identify what program is actually being sent for translation.

If there's a good alternative to logging that meets the same purpose, we should consider that instead.